cumulus/client/parachain-inherent/src/mock.rs

```diff
@@ -230,7 +232,7 @@ impl<R: Send + Sync + GenerateRandomness<u64>> InherentDataProvider
 		sproof_builder.included_para_head = self.current_para_block_head.clone();
 		sproof_builder.num_authorities = 2;
 		let (relay_parent_storage_root, proof, relay_parent_descendants) =
-			sproof_builder.into_state_root_proof_and_descendants(1);
```

Comment:
Hmm how did this work with a value of 1 before, did we break it in previous PR?

Reply:
This was just introduced last week and probably never tested :P